src/System.Linq/samples/AggregateSamples.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+
+namespace System.Linq.Samples
+{
+    public static class AggregateSamples
+    {
+        /// <summary>
+        /// Using Aggregate to reverse a string.
+        /// </summary>
+        public static void Overload1()
+        {
+            string sentence = "there's nothing quite like ice cream";
+
+            string[] words = sentence.Split(' ');
+
+            string reversedSentence = words.Aggregate((workingSentence, nextWord) => nextWord + " " + workingSentence);
+
+            // reversedSentence => "cream ice like quite nothing there's"
+
+            Console.WriteLine(reversedSentence);
+        }
+
+        /// <summary>
+        /// Using Aggregate to sum the lengths of the words in a string.
+        /// </summary>
+        public static void Overload2()
+        {
+            string sentence = "there's nothing quite like ice cream";
+
+            string[] words = sentence.Split(' ');
+
+            int seed = 0;
+            int sumOfWords = words.Aggregate(seed, (workingTotal, nextWord) => workingTotal + nextWord.Length);
+
+            // sumOfWords => 31
+
+            Console.WriteLine($"Sentence: \"{sentence}\"\nHas {sumOfWords} total non-whitepsace characters!");
```

Comment:
Gotcha.  I'll keep it at C# 5 for now then.  Would the solution file change need to be amended as well?  Currently it's been changed to no longer be a vs 2013 solution.